pcmanfm/preferences.ui

```diff
@@ -899,7 +899,7 @@ A value of -1 means that there is no limit for the file size (the default).</str
             <item>
              <widget class="QCheckBox" name="templateTypeOnce">
               <property name="text">
-               <string>Show only one template for each MIME type</string>
+               <string>Show only one template for each media type</string>
```

Comment:
This isn't about media but mime.

Reply:
Unless https://en.wikipedia.org/wiki/MIME then that is all https://en.wikipedia.org/wiki/Media_type

Reply:
→ https://www.freedesktop.org/wiki/Specifications/shared-mime-info-spec/

Reply:
https://www.iana.org/assignments/media-types/media-types.xhtml am I missing something?

Reply:
> am I missing something?

Yes, `freedesktop.org`, which we follow.

Reply:
So it becomes valid here once freedesktop.org updates their wording?

Reply:
There's nothing to update. "image/jpeg is an example of a MIME type where image is the media type, and jpeg is the subtype identifier." (→ https://help.gnome.org/admin/system-admin-guide/stable/mime-types.html.en)

Reply:
Yes, but from my understanding the word to reference the concept changed from "MIME type" to "media type" a while ago, regardless of underlying components.
https://www.iana.org/assignments/media-types/media-types.xhtml was updated 8 days ago.

There is a "top-level media type" and "media subtype" in https://www.rfc-editor.org/rfc/rfc2046.html from 1996